content/docs/rfcs/53/README.md

```diff
@@ -0,0 +1,251 @@
+---
+slug: 53
+title: 53/WAKU2-X3DH
+name: X3DH usage for Waku payload encryption
+status: raw
```

Comment:
It was labelled stable initially, since it is in use by the users of the Status app. as per https://github.com/vacp2p/rfc/pull/569#pullrequestreview-1281254759, was moved to `raw` to indicate lifecycle through vac research. Happy to change it to `draft` though